pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -522,6 +552,273 @@ func containerSucceeded(c *v1.Container, podStatus *kubecontainer.PodStatus) boo
 	return cStatus.ExitCode == 0
 }
 
+func (m *kubeGenericRuntimeManager) computePodResizeAction(pod *v1.Pod, containerIdx int, containerStatus *kubecontainer.Status, changes *podActions) bool {
```

Comment:
@vinaykul 
I just glanced impl of this feature in 10 mins.
(So If I put wrong review, plz clarify it)

I think the idea around this feature is resizing cpu/memory limit then sending CRI update request to CRI runtime to update actual cgroups.

However It seems the changes does not cover `triggering topology manager to re-calculate resources`.
The thing is it will not work with Topology Manager familly.
The topology manager and it's sub features(cpu/memory/device manager) assign resources at the pod admission stage.
(See [the KEP](https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/693-topology-manager) for more details)
CPU/Memory Manager pre-allocate appropriate resources at that stage with {pod id, container id, and resources} then use it when creating real containers.
So the resources allocated at pod admission stage is used when sending CRI update requests.

So why don't you add limitation to this feature like `to use this feature Topology Manager/Cpu Manager/Memory Manager should not be enabled` in `alpha`.
May be we can make changes on TM/CM/MM on the beta stage of this feature.

Reply:
Thanks for catching this. I thought this was called out in the KEP but I don't see it. Perhaps fell through the cracks. I'll capture this in the documentation that goes along with it, and we can address this in beta.

Reply:
@bg-chun Can you please see if the last bullet point in 'non goals' section accurately cover this limitation?
https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/1287-in-place-update-pod-resources#non-goals